stickyuploads/storage.py

```diff
@@ -12,4 +15,18 @@ def __init__(self):
 
     def url(self, name):
         """Files are not accessable via url."""
-        raise NotImplementedError()
\ No newline at end of file
+        raise NotImplementedError()
+
+    def get_available_name(self, name, max_length=None):
+        """Return path to name placed in two random directories"""
+        name = os.path.join(
```

Comment:
I'm not trying to reinvent `mkdtemp`. The purpose of `mkdtemp` is to create unique directory which is not already used.
In our case  we only need to reduce the chance of conflicting file names but still in the same directory can be uploaded more than one file.